views/js/qtiCreator/widgets/interactions/Widget.js

```diff
@@ -87,7 +88,7 @@ define([
     InteractionWidget.createToolbar = function(options){
 
         options = _.defaults(options || {}, {
-            title : _convertToTitle(this.element.qtiClass)
+            title : __(_convertToTitle(this.element.qtiClass))
```

Comment:
Sorry, but this is not how the translation works. We need a static text to be presented as an input for the i18n helper. This is mandatory for the translation scanner to properly gather the text to the PO files.

Using such a dynamic translation will not raise any issue, but no translation will be made since no equivalent sentence does exist in the locale.